### DISCUSSION TITLE
I am working on developing a plugin to manage HTTP, apache, nginx, tomcat and haproxy servers.

### DISCUSSION THREAD

**Discussion by NeftaliYagua** [cockpit-project/cockpit#19733] [Ideas]
I am working on developing a plugin to manage HTTP, apache, nginx, tomcat and haproxy servers.

I am in the brainstorming stage, I have the project hosted at the following address:

https://github.com/cockpit-community/cockpit-webservers

As soon as it is more mature I will request to join it to the cockpit-project, at the moment it is just a sketch.

![Captura desde 2023-12-11 21-19-18](https://github.com/cockpit-project/cockpit/assets/2866454/3c222f6b-995c-4ba3-a07c-6a52f99324f2)

All ideas are welcome, contributions, prototype design collaboration, I will be actively participating in cockpit plugins, I also have one in mind for databases and another for email. I really love cockpit-project and I would love to be a member of the development team.

**Comment by jelly** [comment]
Hi, you are welcome to develop your plugin on GitHub and ask for help in discussions or on our Matrix channel. You don't need to join the Cockpit project to work on a plugin.

**Comment by MaraScott** [comment]
Hi, I'm working on a plugin to manage DB from cockpit and I'm facing this issue https://github.com/MaraScott/cockpit-adminer-manager/issues/3

if someone has any idea how to fix it ? that would help me a lot :)
